apps/typescriptcourse/src/pages/build-a-typescript-project-from-scratch.mdx

```diff
@@ -0,0 +1,75 @@
+import config from '../config'
+import {first} from 'lodash'
+import EmailCourseTemplate from '../templates/email-course'
+
+export const meta = {
+  title: config.defaultTitle,
+  headline:
+    'Build a <span className="bg-gradient-to-l from-sky-50 to-pink-200 bg-clip-text text-transparent decoration-clone">TypeScript Project</span> From Scratch',
+  description: config.description,
+  titleAppendSiteName: false,
+  url: config.siteUrl,
+  ogImage: {
+    url: 'https://typescriptcourse.com/images/emails/build-ts-project-from-scratch/card@2x.png',
+  },
+  formImage: require('../../public/images/emails/build-ts-project-from-scratch/thumb@2x.png'),
+  headerBgClassName: 'bg-landing-header',
+  formBgClassName: 'bg-landing-form',
+  ckFormId: 0, // TODO: add form id
```

Comment:
**💣 chore**: this needs to get updated with correct ck form id that triggers sequence for this second module.

Reply:
Thanks for updating this 🥇